### PR TITLE
Add option to select if source NATing should be enabled or not

### PR DESCRIPTION
**Pull request by Exchizz**
Hi,

This PR implements the required UI changes for this PR: https://github.com/gravitl/netmaker/pull/1399

It looks like this:
image

\Mathias

### CODE REVIEW

**Comment by 0xdcarns** [comment]
@Exchizz would you mind rebasing on develop?

**Comment by Exchizz** [comment]
> @Exchizz would you mind rebasing on develop?

Done :)

File: src/store/modules/node/types.ts [hunk 1]
```diff
@@ -172,7 +172,7 @@ export interface CreateEgressNodePayload {
     payload: {
       ranges: Array<string>
       interface: string
-      natEnabled: boolean
+      natEnabled: string
```
Comment (0xdcarns) [inline src/store/modules/node/types.ts:175]: could make this type more specific -> `"yes" | "no"`

File: src/route/nodes/netid/components/CreateEgress.tsx [hunk 2]
```diff
@@ -106,13 +106,13 @@ export function CreateEgress() {
   interface EgressData {
     ranges: string
     iface: string
-    natEnabled: boolean
+    natEnabled: string
```
Comment (0xdcarns) [inline src/route/nodes/netid/components/CreateEgress.tsx]: could make this type more specific -> `"yes" | "no"`

Comment (Exchizz) [reply src/route/nodes/netid/components/CreateEgress.tsx]: I've changed natEnabled to boolean - it doesn't have to be a string.

**Pull request merged**